res/controllers/midi-components-0.0.js

```diff
@@ -181,9 +181,11 @@
     PlayButton.prototype = new Button({
         unshift: function () {
             this.inKey = 'play';
+            this.onlyOnPress = true;
         },
         shift: function () {
-            this.inKey = 'start_stop';
+            this.inKey = 'reverse';
+            this.onlyOnPress = false;
```

Comment:
The deck might already be playing in reverse when you press shift + play. In this case the deck should continue playing in reverse and continue playing forward when releasing the button. Your implementation toggles reverse both when pressing and releasing the button, which has the opposite effect.